app/Events/S3RestoreJobFinished.php

```
@@ -0,0 +1,56 @@
+            // Clean up any remaining files in database container (may already be cleaned)
+            if (filled($container)) {
+                if (isSafeTmpPath($containerTmpPath)) {
+                    $commands[] = "docker exec {$container} rm -f {$containerTmpPath} 2>/dev/null || true";
+                }
+                if (isSafeTmpPath($scriptPath)) {
+                    $commands[] = "docker exec {$container} rm -f {$scriptPath} 2>/dev/null || true";
+                }
+            }
+
+            if (! empty($commands)) {
+                $server = Server::find($serverId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method '__construct'. (undefined)

(StaticAccess)